### DISCUSSION TITLE
Is development of sk-auth paused?

### DISCUSSION THREAD

**Discussion by asuraphel** [Dan6erbond/sk-auth#102] [General]
First off all, RaviAnand, thanks for creating sk-auth! Are you still interested in maintaining the library or have your priorities changed?

**Comment by Dan6erbond** [comment]
Hi, @asuraphel, thank you for your comment. I have to admit I am aware of the stagnation that the library's development has hit, mostly due to my paid personal projects and job taking over most of the time that I have to work on stuff like this, but it is a project I would like to continue in the future. I will try to comment on some of the latest issues, though, and see if I can get the ball rolling with some secondary contributors to the project.

**Comment by asuraphel** [comment]
@Dan6erbond totally understood. Maintaining auth library for a framework that this rapidly changing is a daunting task! Have you considered asking the Svelte/Kit team or hardcore users on Discord for help?

**Comment by wobsoriano** [comment]
FYI https://authjs.dev/
